contracts/pair/src/tests/setup.rs

```diff
@@ -16,12 +17,19 @@ fn install_token_wasm(env: &Env) -> BytesN<32> {
     env.deployer().upload_contract_wasm(WASM)
 }
 
+fn install_stake_wasm(env: &Env) -> BytesN<32> {
+    soroban_sdk::contractimport!(
+        file = "../../target/wasm32-unknown-unknown/release/phoenix_stake.wasm"
+    );
+    env.deployer().upload_contract_wasm(WASM)
+}
```

Comment:
Since this is now required, modify `Makefile` to require building stake first as well.